RELEASE/scripts/autoscend.ash

```diff
@@ -3125,6 +3128,8 @@ void resetState() {
 
 	horseDefault(); // horsery tracking
 
+	set_property("auto_snapperPhylum", ""); // Red-Nosed Snapper phylum tracking
```

Comment:
I was slightly confused why we needed this when `redSnapperPhylum` exists at first. I see now it's a tentative thing so we don't potentially do multiple changes in server hits and it does nothing until preadv, maybe make the comment mention that?

Reply:
done.